RetailCoder.VBE/Inspections/InspectionBase.cs

```diff
@@ -73,20 +75,56 @@ protected InspectionBase(RubberduckParserState state, CodeInspectionSeverity def
         /// </summary>
         protected virtual IEnumerable<Declaration> Declarations
         {
-            get { return State.AllDeclarations.Where(declaration => !declaration.IsInspectionDisabled(AnnotationName)); }
+            get { return State.AllDeclarations.Where(declaration => !IsInspectionDisabled(declaration, AnnotationName)); }
+        }
+
+        /// <summary>
+        /// Gets all user declarations in the parser state without an @Ignore annotation for this inspection.
+        /// </summary>
+        protected virtual IEnumerable<Declaration> UserDeclarations
+        {
+            get { return State.AllUserDeclarations.Where(declaration => !IsInspectionDisabled(declaration, AnnotationName)); }
```

Comment:
I'd be curious to see any performance between this and [this](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Parsing/VBA/RubberduckParserState.cs#L582-L615)